apps/docs/constants/members.ts

```diff
@@ -0,0 +1,34 @@
+import { AvatarProps } from "../components/Avatar/Avatar.types";
+
+export const members: AvatarProps[] = [
+  {
+    username: "minsoo-web",
+    ghId: "57122180",
+    nameKr: "김민수",
```

Comment:
```suggestion
    username: "minsoo-web",
    githubId: "57122180",
    name: "김민수",
```
`p4` : 요로케 수정은 어떠신가요??

Reply:
username과 헷갈릴까봐 kr이라는 suffix를 붙여봤는데, name도 직관적이네요! 확인입니당

Reply:
[fix: nameKr -> name](https://github.com/function-and-mountain/functional-coding-nutshell/pull/2/commits/7bd3d629c9c22de92f1ac4892380011030a38457)

여기서 작업했습니다~!